testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -112,10 +116,11 @@ public void testTrinoReadingSparkData(StorageFormat storageFormat)
                         ", true" +
                         ", TIMESTAMP '2020-06-28 14:16:00.456'" +
                         ", DATE '1950-06-28'" +
+                        ", CAST('binary0/1values' AS BINARY)" +
```

Comment:
can we have a test on truly binary data? eg not forming valid UTF-8 and eg containing byte 0

eg `byte 0, 1, 2, 3, f0, fe, ff`